code/Arduino/hello-world.ino

```diff
@@ -0,0 +1,14 @@
+// Prathima Kadari - Hello World in Arduino - Display on Liquid Crystal
+
+#include <LiquidCrystal.h>
+
+LiquidCrystal lcd(12, 11, 5, 4, 3, 2);
+
+void setup() {
+   lcd.begin(16, 2);
+   lcd.print("Hello, World!");
+}
+
+void loop() {
+  lcd.setCursor(0, 1);
+  lcd.print(millis() / 1000);
```

Comment:
Hey @prathimacode-hub! I believe you missed a closing curly brace after this line.

Reply:
Oops, yeah. Sorry, didn't realize. Thanks for pointing it out and letting me know. I had corrected it. 👍 @agarwalrounak